content/en/docs/collector/architecture.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Architecture
+weight: 28
+# prettier-ignore
+cSpell:ignore: # TODO: Add keywords to ignore.
+---
+
+OpenTelemetry Collector is an executable that can receive telemetry data,
+optionally process it, and export it further.
+
+The Collector supports several popular open source protocols for receiving and
+sending telemetry data as well as offering a pluggable architecture for adding
+more protocols.
+
+Data receiving, processing, and exporting are done using
+[Pipelines](#pipelines). The Collector can be configured to have one or more
+pipelines. Each pipeline includes the following:
+
+- A set of [Receivers](#receivers) that receive the data.
+- A series of optional [Processors](#processors) that get the data from
+  receivers and process it.
+- A set of [Exporters](#exporters) which get the data from processors and send
+  it outside the Collector.
+
+The same receiver can be included in multiple pipelines and multiple pipelines
+can include the same exporter.
+
+## Pipelines
+
+A pipeline defines a path that data follows in the Collector: from reception, to
+processing (or modification), and finally to export.
+
+Pipelines can operate on three telemetry data types: traces, metrics, and logs.
+The data type is a property of the pipeline defined by its configuration.
+Receivers, processors, and exporters used in a pipeline must support the
+particular data type otherwise `ErrDataTypeIsNotSupported` will be reported when
+the configuration is loaded. A pipeline can be depicted the following way:
+
+```mermaid
+---
+title: Pipeline
```

Comment:
This is the first of six Mermaid diagrams. Mermaid doesn't allow for floating edges, so I've left off the ellipses from the [original diagram](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/images/design-pipelines.png). Hopefully doing so doesn't change the meaning too much.